sway/sway.5.scd

```diff
@@ -186,21 +186,28 @@ set|plus|minus <amount>
 	Cycles the layout mode of the focused container through a list of layouts.
 
 *max_render_time* off|<msec>
-	Works together with *output max_render_time* to reduce the latency even
-	further by delaying the frame callbacks sent to a surface. When set to
-	a positive number of milliseconds, delays the frame callback in such a
-	way that the surface has the specified number of milliseconds to render
-	and commit new contents before being sampled by the compositor for the
-	next presentation. See *max_render_time* in *sway-output*(5) for
-	further details.
+	Controls when the relevant application is told to render this window, as a
+	positive number of milliseconds before next time sway composites the output.
+	A smaller number leads to fresher rendered frames being composited by sway
+	and lower perceived input latency, while a larger number gives the
+	application more time to render and lowers the likelihood of delayed frames.
+
+	When set to off, the relevant application is told to render this window
+	immediately after the output has been composited, to maximize the time
```

Comment:
Does it use the output `max_render_time`? Or are they not related at all?

Reply:
Output `max_render_time` sets when sway composites the output, which view `max_render_time` is relative to.

However, I do see that I have missed an intricacy in view `max_render_time` when set to zero, in which case frame callbacks are sent on display refresh, while if set to a positive value, it is relative to composite.

I'll need to double-check why that is, and add this to the documentation.